dali/operators/generic/constant.h

```diff
@@ -91,6 +91,7 @@ class Constant : public Operator<Backend> {
   void RunImpl(Workspace &ws) override;
```

Comment:
Also, this operator probably requires some more work, I'm not convinced it will work with changing batch size now.

We only fill the output once (so we need to fill it either once in accordance to max_batch_size_, or refill every time it changes).

As AFAIK this operator works by replicating one sample, we probably should be able to share a part of prepared max batch.